backends-common/elasticsearch/src/test/java/org/apache/james/backends/es/EmbeddedElasticSearch.java

```diff
@@ -76,20 +68,10 @@ public void before() {
                 .build())
             .node();
         node.start();
-        awaitForElasticSearch();
     }
 
     @Override
     public void after() {
-        awaitForElasticSearch();
-        try (Client client = node.client()) {
-            client.admin()
-                .indices()
-                .delete(new DeleteIndexRequest(indexName.getValue()))
```

Comment:
what happens to the index ? BTW the commit is not about that, right ?

Reply:
Well:

 - 1. We don't use a single index: we use the mailbox one, and the quota search one. Hence, it makes no sens to clean up a single index.
 - 2. Blowing up ES files is also a vey efficient way of doing the cleanup and that covers this index destruction <3

Reply:
> BTW the commit is not about that, right ?

So doing a ~20 ms useless request has nothing to do about fasting tests? Especially while working with EmbeddedElasticSearch class....

Reply:
The commit says : "Avoid waiting 100ms for nothing upon start and stop, avoid a ~10ms request upon stop."

Sorry but it lacks the explanation you just gave about "I decided to drop the files for a faster cleanup". Just update the comment and it will be ok.

Reply:
Actually it's more:

```
The file were dropped anyway, so the 'partial' DeleteIndex query was useless, I removed it.
```

(because cleanup strategy was to blow up files, I did not change that ;-) )

Reply:
And here we are, the commit comment is updated.